dll/shellext/cabview/precomp.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * PROJECT:     ReactOS CabView Shell Extension
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * COPYRIGHT:   Whindmar Saksit (whindsaks@proton.me)
```

Comment:
I've fixed headers and stuff in a commit https://github.com/reactos/reactos/pull/7494/commits/8ae7e52985edb107e637ffa2a4651620cbbe36a2 (just to avoid posting tons of review comments :)